### PR TITLE
fix(tox): add cargo dependency

### PR DESCRIPTION
**Pull request by jarosevcik**

### CODE REVIEW

File: tox/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 FROM alpine:3
 
 ENV PATH="$PATH:/root/.pyenv/bin:/root/.pyenv/shims"
-ENV version=3.21.2
+ENV version=3.21.3
 
-RUN apk add --no-cache --virtual=.build-deps curl git linux-headers openssl-dev sqlite-dev readline-dev bzip2-dev ncurses-dev sqlite-dev patch xz-dev zlib-dev && \
+RUN apk add --no-cache --virtual=.build-deps curl git linux-headers openssl-dev sqlite-dev readline-dev bzip2-dev ncurses-dev sqlite-dev patch xz-dev zlib-dev cargo && \
```
Comment (aexvir) [inline tox/Dockerfile:6]: nitpick: but can you add it so it's sorted alphabetically like the other packages?

**Review by aexvir — CHANGES_REQUESTED** [review]

File: tox/Dockerfile [hunk 2]
```diff
@@ -1,9 +1,9 @@
 FROM alpine:3
 
 ENV PATH="$PATH:/root/.pyenv/bin:/root/.pyenv/shims"
-ENV version=3.21.2
+ENV version=3.21.3
```
[the remark this answers is not in the document]
Comment (jarosevcik) [reply tox/Dockerfile:4]: then why the version was not bumped also here?

File: tox/Dockerfile (hunk 1 above)
Comment (jarosevcik) [reply tox/Dockerfile:6]: ok, will order also others than :D i checked it and it was not all in alphabetical order so i just added it at the end...but sure, will change

Comment (aexvir) [reply tox/Dockerfile]: damn... 😅️
sorry then
but better now

**Review by aexvir — APPROVED** [review]
Nice!

**Pull request merged**